src/js/themes/micro.js

```diff
@@ -221,44 +221,17 @@ template.innerHTML = html`
     width: 7px;
   }
 
-  media-play-button {
-    display: var(--controls, var(--play-button, inline-flex));
```

Comment:
also note here that I deleted the Mux player "short" CSS vars. 
I figured that was best kept as a specific thing for Mux player.

themes will use the regular display vars defined in Media Chrome. e.g. `--media-seek-backward-button-display`

Mux player adds these short vars with a forwarding like defined here 
https://github.com/muxinc/elements/blob/883a648f9e84a241dc0c4364f8526954b0324429/packages/mux-player/src/styles.css